openlattice.yaml

```diff
@@ -2110,6 +2223,62 @@ paths:
             application/json:
               schema:
                 $ref: "components/DataSearchResult"
+  # /datastore/search/organizations:
```

Comment:
why are these commented out?

Reply:
These are just placeholders.  I wanted to make sure all search-endpoints are documented, or at least in the document, but I don't have time/necessity to write these out :-)